src/systems/ec/specs/components/footer/docs/usage.md

```diff
@@ -1 +1,31 @@
-Do's and don'ts
+The footer appears at the bottom of all  ec.europa.eu pages.
```

Comment:
There are 2 spaces between `all` and `ec.europa.eu`, typo?

Reply:
About branch name convention I apologize, I couldn't find any doc for that (and it was late night so I couldn't ask you). I will follow the pattern that you mention. Also, by mistake I create some branches that you can delete (George-Blockquote-Usage, kibos11-patch-1, kibos11-patch-2).

Reply:
You don't have to apologize for that, there is no documentation nor real convention, but rather commonly used patterns. As long as you don't use parentheses in the branch name, it should be just fine 😉

Reply:
I'll delete the branches you've mentioned

Reply:
It was my experiment with SVGs. Since it's ok to use them then you can merge it I guess.